.gitignore

```diff
@@ -1 +1,2 @@
-.vscode
\ No newline at end of file
+.vscode
+*.sqlite
```

Comment:
you can just do `.sqlite`, that'll exclude everything with that format regardless of filename.
Also include that other bullshit folder you have there.
![https___netflixlife com_files_image-exchange_2021_03_ie_65438-850x560](https://user-images.githubusercontent.com/29634317/112873813-0957d500-90ba-11eb-9644-62543843e622.jpeg)

Reply:
why did you add that picture, it tilts me